.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
+/.buildpath
+/.settings/
```

Comment:
I personally don't mind adding those lines. They hurt no one and make contribution easier for people to come